insights/parsers/satellite_postgresql_query.py

```diff
@@ -182,9 +186,32 @@ class SatelliteComputeResources(SatellitePostgreSQLQuery):
     columns = ['name', 'type']
 
 
+@parser(Specs.satellite_core_taskreservedresource_count)
+class SatelliteCoreTaskReservedResourceCount(SatellitePostgreSQLQuery):
+    """
+    Parse the output of the command ``psql -d pulpcore -c 'select count(*) from core_taskreservedresource' --csv``.
+
+    Sample output::
+
+        count
+        0
+
+    Examples:
+        >>> type(tasks)
+        <class 'insights.parsers.satellite_postgresql_query.SatelliteCoreTaskReservedResourceCount'>
+        >>> tasks[0]['count']
+        '0'
+    """
+    columns = ['count']
+
+
 @parser(Specs.satellite_katello_empty_url_repositories)
 class SatelliteKatelloEmptyURLRepositories(SatellitePostgreSQLQuery):
     """
+    .. warning::
```

Comment:
Please use https://insights-core.readthedocs.io/en/latest/api.html?highlight=deprecate#feature-deprecation

Reply:
@psachin, thanks for the review. I update it, please help to review again.